docker-compose.yml

```diff
@@ -60,6 +60,29 @@ services:
     networks:
       - backend-net
 
+  keycloak:
+    container_name: keycloak
+    image: docker.io/bitnami/keycloak:15
+    environment:
+      - KEYCLOAK_USER=admin
+      - KEYCLOAK_PASSWORD=admin
+      - DEBUG=true
+      - DEBUG_PORT='*:8787'
+      - DB_VENDOR=POSTGRES
+      - DB_ADDR=pg-db
```

Comment:
I'm not sure it's a good idea to reuse the same db for backend and keycloak

Let's make another db instance dedicated to keycloak